DoctorWho2/Controllers/EpisodeController.cs

```diff
@@ -50,6 +50,23 @@ public async Task<ActionResult<int>> CreateEpisode(EpisodeForCreateDto episode)
 
             return Ok(EpisodeEntity.EpisodeId);
         }
+        [HttpPost]
+        [Route("{episodeId}/enemy/")]
+        public async Task<ActionResult<Enemy>> AddEnemyToEpisodeAsync(
+         int episodeId ,EnemyForCreateDto enemyToAdd)
+        {
+            var enemyEntity = _mapper.Map<Enemy>(enemyToAdd);
+            var IsAdd = await _episodeRepository.AddEnemyToEpisodeAsync(enemyEntity ,episodeId);
+            if( IsAdd == false )
+            {
+                return BadRequest("Episode doesn't exist");
+            }
+            await _episodeRepository.SaveChanges();
+            EnemyForCreateDto result = _mapper.Map<EnemyForCreateDto>(enemyEntity);
+            return Ok(result);
+
+        }
+
```

Comment:
the req in exercise is : 8. Add `AddEnemyToEpisode` API.

this is my endpoint : https://localhost:5001/api/Episodes/{enemyId}/enemy/
in body :enemyforcreateDto

it's right ,right?